### PR TITLE
parsers: allow or disallow unknown URL args

### PR DESCRIPTION
**Pull request by fenekku**
Needed for aggregation work.

Not available in webargs 5.x < 6.x . See https://webargs.readthedocs.io/en/latest/upgrading.html#data-is-not-filtered-before-being-passed-to-schemas-and-it-may-be-proxified

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -36,6 +36,7 @@ before_install:
 install:
   - "travis_retry pip install -r .travis-${REQUIREMENTS}-requirements.txt"
   - "travis_retry pip install -e .[all]"
+  - pip freeze
```
Comment (zzacharo) [inline .travis.yml:39]: minor: Out of curiosity, why is that needed?

**Review by zzacharo — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (fenekku) [reply .travis.yml:39]: It's useful for debugging Travis in general: it lists the packages that are installed. 

Here for instance, I was able to see that Black 20.8b1 was installed on Travis while I am running Back 19.10b0 locally which then explained why the Black tests worked locally but not on Travis. 20.8b1 changed how it deals with trailing commas...

Comment (zzacharo) [reply .travis.yml:39]: Ah ok I see! I knew the command but I didn't think that use case :P Indeed is really useful ;)
